specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -3779,6 +3789,34 @@
         }
       }
     },
+    "MetaDataContract": {
+      "type": "object",
+      "description": "A combination of user defined name and user specified data and recognition model name for largePersonGroup/personGroup, and largeFaceList/faceList.",
+      "properties": {
+        "recognitionModel": {
+          "$ref": "#/definitions/RecognitionModel"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/NameAndUserDataContract"
+        }
+      ]
+    },
+    "RecognitionModel": {
+      "type": "string",
+      "description": "Recognition model name. Recognition model is used when the face features are extracted and associated with (Large)FaceList or (Large)PersonGroup. So a recognition model version could be provided when performing a Detection and create (Large)FaceList or (Large)PersonGroup create. If the user does not provide recognition model, default is 'recognition_v01'.",
```

Comment:
"description": "Name of recognition model. Recognition model is used when the face features are extracted and associated with detected faceIds, (Large)FaceList or (Large)PersonGroup. A recognition model name can be provided when performing Face - Detect or (Large)FaceList - Create or (Large)PersonGroup - Create. The default value is 'recognition_v01', if latest model needed, please explicitly specify the model you need.",

Reply:
Done.